### ISSUE TITLE
[Dialog] not client-safe

### ISSUE THREAD

**Issue by DevChannel-de** [Scrounger/ioBroker.vis-materialdesign#10]
Currently the display of a dialog depends on an object ID. Because object IDs are not separated by each client (are not client-safe), it leads to the behavior that when one user opens a dialog other users get it displayed automatically in parallel even if they don't want to see that (usually... ;-)).

As a potential way to solve it: other widgets solve it by providing the trigger element (button, link) together with the dialog so that the open/closed state can be controlled internally, client-safe.

**Comment by Scrounger** [comment]
Ok, but i dont need this feature, so it has very low priority.
And this will break the feature to place the dialog on any position via editor.

**Comment by DevChannel-de** [comment]
Thanks for keeping it open, and of course it's matter of prio ;-)
I want only to bring up that already two users using the same UI could experience the behavior  erroneously because they will be "disrupted" by opened dialogs of each other.

**Comment by SebastianSchultz** [comment]
I want to vote for this :) Because it has prio for me :)

**Comment by gaga191281** [comment]
**+1**. I am going to use 11 Tablets in my house to control lights, blinds, etc.
The only way to work around this "issue" would be to make a own project for each instance. That would not be very effective.

**Comment by losnoxos** [comment]
Da schließe ich mich an. Für die breite Masse dürfte das Prio 1 sein.
Habe grade angefangen mein Vis auf dein Design umzubauen. 
Bei nur einer möglichen Instanz ist es für mich leider nicht brauchbar. 

Das Vis wird von mehreren Benutzer genutzt und läuft auch durchgehend an 3D Druckern usw.

**Comment by Scrounger** [comment]
das neue Dialog Widget kann beides jetzt, button oder datenpunkt.
Enthalten ab d2e29e24090189eea13dde4c73b4866c2664800b
